mapping/server_types.json

```
@@ -12,7 +10,18 @@
     "160sk22": "KS-2C",
     "160sk23": "KS-2D",
     "161sk2": "KS-2E",
-    "160sk1": "KS-1",
+    "1804sk12": "KS-1",
```

by the time i'm writing this, KS-1 is referred by 1801sk12, not 1804sk12 as this pull request suggests.

```suggestion
    "1801sk12": "KS-1",
```